### ISSUE TITLE
Databases

### ISSUE THREAD

**Issue by Koruy** [Divested-Mobile/hypatia#80]
Databases are not updated
![Screenshot_20240504_131406](https://github.com/Divested-Mobile/Hypatia/assets/53151866/f0310591-e6e1-402d-9c35-32939d13d48a)

**Comment by SkewedZeppelin** [comment]
It looks like you're using a VPN.
Please ensure it supports IPv6 and is enabled.

**Comment by Koruy** [comment]
Hypatia app traffic is not included in VPN filtering
![Screenshot_20240504_175425](https://github.com/Divested-Mobile/Hypatia/assets/53151866/19f9d958-ccf2-4f8c-a23c-c0732185035a)

**Comment by Koruy** [comment]
Scanning is not performed, the database is not accessible
![Screenshot_20240504_234141](https://github.com/Divested-Mobile/Hypatia/assets/53151866/bc13ef0b-3dcf-4aae-82bb-ef062c8f3820)

**Comment by lucasmz-dev** [comment]
@SkewedZeppelin Hold on, is Hypatia's database server IPv6 only? :)

**Comment by SkewedZeppelin** [comment]
@lucasmz-dev
no, but many IPv4 addresses are blocked via https://github.com/divestedcg/scfw3 to manage resources

**Comment by Koruy** [comment]
It is not updated via wifi, but it was possible to update via the mobile Internet.  This is strange, just a week ago everything was updated via wifi.
![Screenshot_20240505_054054](https://github.com/Divested-Mobile/Hypatia/assets/53151866/69d10e91-df1f-4d5b-b183-04634bde557a)

**Comment by Koruy** [comment]
Database update ok - this provider is weird beltelecom

**Comment by SkewedZeppelin** [comment]
A new Hypatia update has been released which adds a mirror preset you can use in the `database server override` option.

This mirror server should be accessible.
